src/components/Logbar/Logbar.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { BsPersonCircle } from "react-icons/bs";
+import { BsPencilSquare } from "react-icons/bs";
+import { BsLockFill } from "react-icons/bs";
+
+import "./Logbar.scss";
+
+const Logbar = () => {
+  const [selectedOption, setSelectedOption] = useState("login");
+
+  const handleOptionChange = (event) => {
+    setSelectedOption(event.target.value);
+  };
+
+  return (
+    <div className="logBarMainContainer">
+      <label>
+        <input
+          type="radio"
+          name="option"
+          value="login"
+          checked={selectedOption === "login"}
+          onChange={handleOptionChange}
+        />
+        <span className="checkmark">
```

Comment:
Not descriptive enough, checkmark of what? Element? Page?

Reply:
changed on logCheckmark